apis/python/operator/src/lib.rs

```diff
@@ -110,7 +110,13 @@ impl PyEvent {
             pydict.insert("_cleanup", cleanup.into_py(py));
         }
 
-        Ok(pydict.into_py_dict_bound(py).unbind())
+        Ok(match pydict.into_py_dict(py) {
+            Ok(dict) => dict,
+            Err(e) => {
+                return Err(eyre!("Failed to create Python dictionary: {}", e).into());
+            }
+        }
```

Comment:
```suggestion
        Ok(pydict.into_py_dict(py)?
```

For readibility let's just use the ? sign on all match `.into_py_dict`